TCL/polarDensity_for_DTA.tcl

```diff
@@ -320,7 +302,6 @@ proc local_mid_plane2 {atsel_in frame_i} {
     }
 }
 
-
 proc leaflet_sorter_0 {atsel_in head tail frame_i} {
```

Comment:
The atomselections are deleted after we return (the `$sel delete` lines never get executed)

Reply:
Should be fixed now